### DISCUSSION TITLE
[FEAT] Add JXL as a -compress type

### DISCUSSION THREAD

**Discussion by AngelaDMerkel** [ImageMagick/ImageMagick#7759] [Development] [answered]
### ImageMagick version

7.1.1-39

### Operating system, version and so on

macOS

### Description

JP2 and JPEG compression are already offered within TIFF as compression options, but the container would benefit from being able to utilise JXL compression. The TIFF standard already has a tag which supports JXL as foreign compression.

**Comment by fmw42** [comment, marked as answer]
Imagemagick uses libtiff for reading and writing tiff. This request should be directed to the libtiff team, not Imagemagick.

**Reply by AngelaDMerkel** [reply]
Forgive me for reopening after I have closed the issue, but how can I direct this to the libtiff team? There is no clear pathway for feature requests in github or gitlab

**Reply by fmw42** [reply]
Search google for "libtiff". Find their website and ask there
